README.md

```diff
@@ -62,28 +63,13 @@ it is expected that the imported class was defined correctly, in another file, a
       .factory('someFactory', someFactory);
   ```
 
-  The same components are now separated into their own files.
+  The same directives are now separated into their own files.
 
-  ```js
```

Comment:
Something looks awry here... Diff is showing a lot more edits than I recall making?  I will clean this up and push up a revised document...